src/backends/policy_engine/souffle/dp_testdata/dp_analysis.ir

```diff
@@ -0,0 +1,13 @@
+module m0 {
+  block b0 {
+        %0 = core.input [name: "revenue_table"] ()  
+        %epsilon = global_epsilon [epsilon: "2"] ()
+        %1 = sql.filter [column: "revenue"] (%0.out) // { revenue > 100 }
```

Comment:
To avoid tests from failing because of "Value not found" error, please change %0.out to %0. Similarly in all other places below.

Reply:
done.